### PR TITLE
Review before self-sign

### PR DESCRIPTION
**Pull request by wagnerand**
Request review before self-signing.

### CODE REVIEW

**Comment by wagnerand** [comment]
~Hm, for some reason the bug update doesn't happen after signing. I am not sure if it's related to this patch.~

File: src/mozblocklist.js [hunk 1]
```diff
@@ -285,6 +285,16 @@ export default class Mozblocklist {
...
+    console.warn("Signing blocklist...");
+    if (selfsign) {
+      if (await this.kinto.getBlocklistStatus() === "work-in-progress") {
+        await this.kinto.reviewBlocklist();
+      }
+      await this.kintoapprover.signBlocklist();
+    } else {
+      await this.kinto.signBlocklist();
+    }
+
```
Comment (wagnerand) [inline src/mozblocklist.js:300]: Moving this block up makes sure we put things on _preview_ first (if there is anything to stage), and afterwards fetch the preview.

Comment (kewisch) [reply src/mozblocklist.js:300]: This doesn't cover the case where pending is passed in, which happens for reviewAndSignBlocklist. I also wanted to move the question to remove the security group up front so you don't have to wait so long to answer the question.

Comment (kewisch) [reply src/mozblocklist.js:300]: Also, wouldn't this make `res = []` when the block is self-signed and `pending = null` ?

Comment (wagnerand) [reply src/mozblocklist.js:300]: Should be fixed in d611cb7ff9724d98b4114f65813fe236b9b12309.

**Comment by kewisch** [comment]
I ended up fixing this slightly differently, sorry for the double work. I should have mentioned this before I started to do the work but I just kind of got into it while trying to clear the list of open bugs :-}

c275e850cdd5a6b8785cf1eea2f4272d38bb5157 (I'm aware I messed up with the test command :)

**Pull request closed without merging**